src/Adapter/PdoAdapter.php

```diff
@@ -41,6 +48,13 @@ public function __construct(string $driver, array $options, string $role)
         $this->options = $options;
     }
 
+    public function supportNamedParameterWithArrayValue(): void
```

Comment:
I would do a shorter name. Maybe `enableArrayValues` or something.

Reply:
I used a slightly diff name: `enableNamedArrayValues()` to make it clear. reminder, unnamed placeholders are not supported.